src/plots/gl2d/scene2d.js

```diff
@@ -137,6 +137,11 @@ proto.makeFramework = function() {
     canvas.style.left = '0px';
     canvas.style['pointer-events'] = 'none';
 
+    // disabling user select on the canvas
+    // sanitizes double-clicks interactions
+    // ref: https://github.com/plotly/plotly.js/issues/744
+    canvas.className += 'user-select-none';
```

Comment:
@dfcreative I chose to add the `'user-select-none'` class to the gl2d `<canvas>` nodes only to be safe - that way this patch won't affect the other plot types that use the `gl-container` div.